app/javascript/dashboard/helper/filterQueryGenerator.js

```diff
@@ -11,7 +11,7 @@ const generatePayload = data => {
     }
     return item;
   });
-
+  payload[payload.length - 1].query_operator = null;
```

Comment:
```suggestion
// For every query added, the query_operator is set default to and so the
// last query will have an extra query_operator, this would break the api. 
// Setting this to null for all query payload

  payload[payload.length - 1].query_operator = null;
```